Source/UI/NewUI/BillingForm/View/BillingFormPhoneNumberText.swift

```diff
@@ -1,101 +1,31 @@
+import Checkout
 import UIKit
 import PhoneNumberKit
 
 protocol BillingFormPhoneNumberTextDelegate {
-    func updateCountryCode(code:Int)
-    func validateNumber()
+    func phoneNumberIsUpdated(number: String?)
 }
 
-final class BillingFormPhoneNumberText: BillingFormTextField {
-
-    // MARK: - Properties
+final class BillingFormPhoneNumberText: PhoneNumberTextField, BillingFormTextField {
```

Comment:
I get multiple inheritance or conformance, but multiple inheritances that seem to be similar does add to confusion. Why PhoneNumberTextField and BillingFormTextField ? Are they so different they need to be different things? This additional complexity is very confusing imo and would mean a dev would have to fork out and navigate multiple paths now to understand what's going on from here.

Reply:
- `PhoneNumberTextField` is the parent textfield from `PhoneNumberKit`
- `BillingFormPhoneNumberText` is a `PhoneNumberTextField` and implement `BillingFormTextField` protocol